fractal_server/app/models/security.py

```diff
@@ -66,7 +68,7 @@ class UserOAuth(SQLModel, table=True):
 
     oauth_accounts: list["OAuthAccount"] = Relationship(
         back_populates="user",
-        sa_relationship_kwargs={"lazy": "selectin", "cascade": "all, delete"},
+        sa_relationship_kwargs={"lazy": "joined", "cascade": "all, delete"},
```

Comment:
Let's make sure we are OK with this change (cc @mfranzon).

For the record, this `joined` matches with https://fastapi-users.github.io/fastapi-users/11.0/configuration/oauth/#sqlalchemy_1, where
```python

class OAuthAccount(SQLAlchemyBaseOAuthAccountTableUUID, Base):
    pass


class User(SQLAlchemyBaseUserTableUUID, Base):
    oauth_accounts: Mapped[List[OAuthAccount]] = relationship(
        "OAuthAccount", lazy="joined"
    )
```

Reply:
- selectin: https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#selectin-eager-loading
- joined: https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#joined-eager-loading